examples/server/tests/unit/test_embedding.py

```diff
@@ -45,19 +47,49 @@ def test_embedding_multiple():
         assert len(d['embedding']) > 1
 
 
+def test_embedding_pooling_none():
+    global server
+    server.pooling = 'none'
+    server.start()
+    res = server.make_request("POST", "/embeddings", data={
+        "input": "hello hello hello",
+    })
+    assert res.status_code == 200
+    assert 'embedding' in res.body[0]
+    assert len(res.body[0]['embedding']) == 3
+
+    # make sure embedding vector is not normalized
+    for x in res.body[0]['embedding']:
+        assert abs(sum([x ** 2 for x in x]) - 1) > EPSILON
+
+
+def test_embedding_pooling_none_oai():
+    global server
+    server.pooling = 'none'
+    server.start()
+    res = server.make_request("POST", "/v1/embeddings", data={
+        "input": "hello hello hello",
+    })
+
+    # /v1/embeddings does not support pooling type 'none'
+    assert res.status_code == 400
+
+
 def test_embedding_openai_library_single():
     global server
+    server.pooling = 'last'
     server.start()
-    client = OpenAI(api_key="dummy", base_url=f"http://{server.server_host}:{server.server_port}")
+    client = OpenAI(api_key="dummy", base_url=f"http://{server.server_host}:{server.server_port}/v1")
     res = client.embeddings.create(model="text-embedding-3-small", input="I believe the meaning of life is")
     assert len(res.data) == 1
     assert len(res.data[0].embedding) > 1
 
 
 def test_embedding_openai_library_multiple():
     global server
+    server.pooling = 'last'
     server.start()
-    client = OpenAI(api_key="dummy", base_url=f"http://{server.server_host}:{server.server_port}")
+    client = OpenAI(api_key="dummy", base_url=f"http://{server.server_host}:{server.server_port}/v1")
```

Comment:
I had to add the `/v1` suffix for these calls, otherwise the client will hit the `/embeddings` endpoint which is not OAI compatible. Is this the correct fix?

Reply:
Yes that's correct, I haven't notice that. The original OAI code has `/v1` in its default value: https://github.com/openai/openai-python/blob/e94d98e9bf97a5d2d02d79d58f2abdbab26ff2bd/src/openai/_client.py#L117

We should change all other recurrence too (in another PR)

Reply:
btw for embeddings endpoint we also have the switch between non-OAI and OAI using the presence of `content` and `input` fields in the response. Is it still relevant now?

Reply:
I've modified this to be able to use both endpoints with both inputs. And I'm thinking to also make the change that `content` is alias to `input` - maybe we can do this directly in #10866